build.gradle

```diff
@@ -80,6 +82,15 @@ pluginBundle {
 // in the TestKit plugin classpath.
 def localRepo = file("$buildDir/local-repo")
 publishing {
+    publications {
+        maven(MavenPublication) {
```

Comment:
Can you add a comment here if we can remove this section in the future when the plugin portal automatically supports publishing the signature files? That way we aren't wondering if this should be removed.

Reply:
done